db/migrate/20230401123456_add_new_columns_to_users.rb

```diff
@@ -0,0 +1,8 @@
+class AddNewColumnsToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :vip, :boolean, default: false
+    add_column :users, :role, :string
+    add_column :users, :team_member_id, :bigint
```

Comment:
Adding a new column 'role' to the users table without any validation can lead to inconsistent data. Consider adding validations for the 'role' field to ensure that only allowed roles can be assigned to users.